src/components/posts/Post.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Post = ({ post }) => {  
+  return (
+    <>
+    <li>
+      <h2>{post.userId}</h2>
+      <h3>{post.title}</h3>
+      <h4>{post.body}</h4>
```

Comment:
```suggestion
      <p>{post.body}</p>
```
Text content is ideally wrapped in a p tag -- use CSS to achieve style you want.